src/utils.ts

```diff
@@ -32,17 +32,18 @@ export const pluginRootDir: PortablePath =
  * @param {boolean} recursive - Whether to compute licenses recursively
  * @param {boolean} production - Whether to exclude devDependencies
  * @param {boolean} excludeMetadata - Whether to exclude metadata in tree
- * @returns {treeUtils.TreeNode} Root tree node
+ * @returns Root tree node
  */
 export const getTree = async (
   project: Project,
   json: boolean,
   recursive: boolean,
   production: boolean,
   excludeMetadata: boolean
-): Promise<treeUtils.TreeNode> => {
-  const rootChildren: treeUtils.TreeMap = {}
-  const root: treeUtils.TreeNode = { children: rootChildren }
+) => {
+  const root = {
```

Comment:
is this a functional change? or a style thing?

Reply:
Type asserting it as `TreeMap` meant that children could be another `TreeMap`, an array of `TreeNode` or `false`, which created some ugly code to check later on. However, I just quickly put that and will improve this code, probably with `satisfies`